siad/daemon.go

```diff
@@ -77,16 +77,32 @@ func processModules(modules string) (string, error) {
 	return modules, nil
 }
 
+// processProfiles checks that the flags given for profiling are valid.
+func processProfile(profile string) (string, error) {
```

Comment:
docstring and function have a different name, and a better name for this is probably proceesProfileFlags